graph/src/data/subgraph/mod.rs

```diff
@@ -956,6 +1022,10 @@ impl<C: Blockchain> SubgraphManifest<C> {
             }),
         }
     }
+
+    pub fn unified_mapping_api_version(&self) -> UnifiedMappingApiVersion {
+        self.mappings().iter().collect()
+    }
```

Comment:
Iterating over `mappings()` so we get all values from both `data_sources` and `templates`